MultiSigWalletOverride/migrations/2_deploy_contracts.js

```diff
@@ -4,25 +4,33 @@ const MultisigWalletFactory = artifacts.require('MultiSigWalletWithDailyLimitFac
 const fs = require('fs');
 const path = require('path');
 
-module.exports = deployer => {
+module.exports = async (deployer, [_deployerAddress]) => {
   const args = process.argv.slice();
-  if (process.env.DEPLOY_FACTORY){
-    deployer.deploy(MultisigWalletFactory);
-    console.log("Factory with Daily Limit deployed");
-  } else if (args.length < 5) {
-    console.error("Multisig with daily limit requires to pass owner " +
-      "list, required confirmations and daily limit");
-  } else if (args.length < 6) {
-    deployer.deploy(MultisigWalletWithoutDailyLimit, args[3].split(","), args[4]).then(function() {
+  let accountsIndex = undefined;
```

Comment:
probably you can use `indexOf` function here.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf